api.raml

```diff
@@ -129,7 +129,7 @@ resourceTypes:
     securedBy: [oauth_2_0: {scopes: ["manage_project:{projectKey}"]}]
     body:
       application/json:
-        type: Update
+        type: ProjectUpdate
```

Comment:
atm we should not do this as not all tools correctly handle the discriminators. Type specific updates are set in the update-actions.raml